app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def show
+    @post = Post.find_by(id: params[:id])
+  end
```

Comment:
- [ ] good job adding the post index file in the views, however, as the previous CR already mentioned there should be an index action in the post controller as well. please ensure this is included as well.